routers/issueRouter.js

```diff
@@ -42,4 +42,16 @@ router.post( '/', jsonParser, function( req, res ){
 	});
 });
 
-module.exports = router;
+// PUT issue, updating a current issue
+router.put( '/', jsonParser, function( req, res){
+	var issue = req.body;
+	db.findByIdAndUpdate(issue._id, { name: issue.name, description: issue.description, location: { lat: issue.location[lat], long: issue.location[long]}, priority: issue.priority, open: issue.open, reportedDate: issue.reportedDate, resolvedDate: issue.resolvedDate}, function( err, doc ){
```

Comment:
we can just pass in the schema object? I was under the assumption that you had to construct that in the findbyidandupdate call. I can make that change